### DISCUSSION TITLE
第6节课作业(5班)

### DISCUSSION THREAD

**Discussion by GitEasonXu** [InternLM/Tutorial#446] [作业]
基础作业
- 使用 OpenCompass 评测 InternLM2-Chat-7B 模型在 C-Eval 数据集上的性能

进阶作业
- 使用 OpenCompass 评测 InternLM2-Chat-7B 模型使用 LMDeploy 0.2.0 部署后在 C-Eval 数据集上的性能

**Comment by GitEasonXu** [comment]
作业：https://github.com/GitEasonXu/InternLM/blob/main/homework/course_6/coursework.md

**Comment by angellyao** [comment]
https://zhuanlan.zhihu.com/p/678917434

**Comment by ZhaoHongSen0726** [comment]
https://blog.csdn.net/m0_75085274/article/details/135727946?spm=1001.2014.3001.5502

**Comment by JayceNing** [comment]
**基础作业**
使用 OpenCompass 评测 InternLM2-Chat-7B 模型在 C-Eval 数据集上的性能

评测运行代码
``` python
python run.py --datasets ceval_gen --hf-path /root/model/Shanghai_AI_Laboratory/internlm2-chat-7b/ --tokenizer-path /root/model/Shanghai_AI_Laboratory/internlm2-chat-7b/ --tokenizer-kwargs padding_side='left' truncation='left' trust_remote_code=True --model-kwargs trust_remote_code=True device_map='auto' --max-seq-len 2048 --max-out-len 16 --batch-size 4 --num-gpus 1 --debug
```
评测结果
![image](https://github.com/InternLM/tutorial/assets/44694099/cb9645f4-7d7d-4652-b6d3-2e62b0d6621a)
![image](https://github.com/InternLM/tutorial/assets/44694099/048dae7a-b52a-4b18-8549-31a8abde985e)

**Reply by JayceNing** [reply]
![image](https://github.com/InternLM/tutorial/assets/44694099/5cf5af51-8f62-42c0-9da9-b11a620f06a6)
![image](https://github.com/InternLM/tutorial/assets/44694099/08430fb9-1b51-43ab-99d6-b06e99039ad6)

**Comment by Farewell-CK** [comment]
https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E5%85%AD%E8%8A%82%E8%AF%BE/note.md

**Comment by Farewell-CK** [comment]
https://github.com/8baby8/Farewell_Learn_puyu/blob/main/%E7%AC%AC%E5%85%AD%E8%8A%82%E8%AF%BE/task.md

**Comment by shengmouget** [comment]
[作业](https://github.com/shengmouget/LLM_Job?tab=readme-ov-file#%E7%AC%AC%E4%BA%94%E6%AC%A1%E4%BD%9C%E4%B8%9A)

**Comment by ATang0729** [comment]
https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class6-HW/class6_HW.md

**Comment by sanbuphy** [comment]
https://zhuanlan.zhihu.com/p/680168238?

**Comment by Milkve** [comment]
[第六节课作业](https://juejin.cn/post/7329753721017597967)
